src/components/clusterConfiguration/DiscoveryImageTypeControlGroup.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Split, SplitItem } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+import { PopoverIcon } from '../ui';
+import { RadioField } from '../ui/formik/RadioField';
+
+const GROUP_NAME = 'imageType';
+
+const DiscoveryImageTypeControlGroupLabel = ({
+  text,
+  popoverContent,
+}: {
+  text: string;
+  popoverContent: string;
+}) => (
+  <>
+    {text}
+    <PopoverIcon
+      component={'a'}
+      variant={'plain'}
+      IconComponent={HelpIcon}
+      bodyContent={popoverContent}
+    />
+  </>
+);
+
+const DiscoveryImageTypeControlGroup = () => (
+  <Split hasGutter>
+    <SplitItem>
+      <RadioField
+        name={GROUP_NAME}
+        id={'full-iso'}
+        value={'full-iso'}
+        label={
+          <DiscoveryImageTypeControlGroupLabel
+            text={'Full ISO'}
+            popoverContent={
+              'Use when you plan to boot your host by attaching the ISO with virtual media. ' +
+              'Minimal ISO contains a smaller image size, and only part of the image will ' +
+              'download on the first boot.'
+            }
+          />
+        }
+      />
+    </SplitItem>
+    <SplitItem style={{ marginRight: '8.2rem' }} />
```

Comment:
Can we use classes?

Reply:
The best PF class I could find, look like this:
![image](https://user-images.githubusercontent.com/77008341/121504296-938db500-c9ea-11eb-873d-eecb822869e8.png)

Reply:
Looks good to my eyes. I would with it unless @nirfarkas thinks differently.